modules/core/src/lib/test/TestStepsWithOptions.ts

```diff
@@ -10,7 +10,7 @@ const TestStepsWithOptions: IExtensionConstructor = class TestStepsWithOptions i
   options = {
     EXISTS: {
       desc: 'option exists',
-      parse: (input: string) => 42,
+      parse: (input: string) => ({ result: 42 }),
```

Comment:
although the answer to the ultimate question of life, the universe and everything is 42...

`'input' is declared but its value is never read.`